packages/dashboard/src/app/serializers/test/templates.js

```diff
@@ -41,13 +43,60 @@ describe('reshapeTemplateObject', () => {
     version: 17,
   };
 
+  it('should return object', () => {
```

Comment:
Not really a problem but maybe we should be a little more descriptive?
```suggestion
  it('should return the template object', () => {
```